src/certificates.cc

```diff
@@ -12,12 +12,63 @@
  * permissions and limitations under the License.
  */
 
+#include <cstdint>
+#include <iostream>
 #include <type_traits>
 
 #include "certificates.h"
+#include "util/strings.h"
 
 namespace zdb {
 
+namespace {
+
+// TODO: Write function to apply a predicate to every CTServerStatus or
+// RootStoreStatus, with the goal of reducing duplicated code in the
+// certificate_has_X functions.
+
+}  // namespace
+
+std::string translate_certificate_source(int source) {
```

Comment:
This should probably have everything from https://github.com/Censys/censys-definitions/blob/master/proto/certificate.proto#L18 in it.